src/containers/LoginScreen/index.js

```diff
@@ -40,4 +40,4 @@ const mapDispatch = dispatch => ({
   login: user => dispatch(login(user))
 });
 
-export default connect(null, mapDispatch)(LoginScreen);
+export default connect(null, mapDispatch)(withNavigation(LoginScreen));
```

Comment:
Wouldn't you think that we could place these `withNavigation` calls in `src/screens/index.js` instead of the container? 

Maybe it doesn't really make a difference, thought of that to not keep wrapping containers here, and thinking that most of registered screens there will need it, if not all.

Reply:
Will do.
Regarding your other question, i thought we could user this context for more that just navigation, that's why i called it screenContext.